tst/Carnival/Unit/Data/Topics/ExampleTopic.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Tests\Carnival\Unit\Data\Topics;
```

Comment:
Regarding your addition of the `Unit` directory to the path for tests, the problem was with the testsuite directory in `phpunit.xml`. I've fixed it in my branch. Recommend you do the same here:

```xml
<testsuite name="Unit">
    <directory suffix="Test.php">./tst/Carnival/</directory>
</testsuite>
```

Reply:
Can do!